<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GayfloroGUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../.designer/backup/res/img/chadThink.png</normaloff>../../.designer/backup/res/img/chadThink.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget {background:#252525}
QTextEdit {
background:#353535;
color:white;
margin:-1px
}

QTabWidget::pane{background:#353535}

QTabBar::tab{
background:#252525;
padding:8;
min-width:9ex;
color:white;
text-align: center;

}
QTabBar::tab:selected {
background:#353535;
}
QPushButton {
background:#353535;
color:white;
}
QPushButton:!enabled {
color:grey;
}
QLineEdit {
background:#353535;
color:white;
border:#424242;
}
</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>921</width>
      <height>381</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tab_Main">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QTextEdit" name="console_main">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>921</width>
        <height>354</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Debug">
     <attribute name="title">
      <string>Debug</string>
     </attribute>
     <widget class="QTextEdit" name="console_debug">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>921</width>
        <height>354</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_sonfloro">
     <attribute name="title">
      <string>#sonfloro</string>
     </attribute>
     <widget class="QTextEdit" name="console_sonfloro">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>921</width>
        <height>354</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>53</red>
            <green>53</green>
            <blue>53</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>53</red>
            <green>53</green>
            <blue>53</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>53</red>
            <green>53</green>
            <blue>53</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="NoBrush">
           <color alpha="128">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>53</red>
            <green>53</green>
            <blue>53</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>53</red>
            <green>53</green>
            <blue>53</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>53</red>
            <green>53</green>
            <blue>53</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="NoBrush">
           <color alpha="128">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>53</red>
            <green>53</green>
            <blue>53</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>53</red>
            <green>53</green>
            <blue>53</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>53</red>
            <green>53</green>
            <blue>53</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="NoBrush">
           <color alpha="128">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_zoey">
     <attribute name="title">
      <string>#iTzMissZoey</string>
     </attribute>
     <widget class="QTextEdit" name="console_zoey">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>921</width>
        <height>354</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_lumiriss">
     <attribute name="title">
      <string>#lumiriss</string>
     </attribute>
     <widget class="QTextEdit" name="console_lumiriss">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>921</width>
        <height>354</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_whatthehellshappened">
     <attribute name="title">
      <string>#whatthehellshappened</string>
     </attribute>
     <widget class="QTextEdit" name="console_whatthehellshappened">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>921</width>
        <height>354</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_armorysons">
     <attribute name="title">
      <string>#armorysons</string>
     </attribute>
     <widget class="QTextEdit" name="console_armorysons">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>921</width>
        <height>354</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_realmcopier">
     <attribute name="title">
      <string>#realmcopier</string>
     </attribute>
     <widget class="QTextEdit" name="console_realmcopier">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>921</width>
        <height>354</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>681</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="chatLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chatButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Chat</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionTesting">
   <property name="text">
    <string>Testing</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
